source/codegen/metadata/niswitch/functions.py

```diff
@@ -443,6 +461,8 @@
     },
     'ErrorMessage': {
         'cname': 'niSwitch_error_message',
+        'codegen_method': 'public',
+        'is_error_handling': True,
```

Comment:
Hmm, little surprised at the addition  of 'is_error_handling' here and a few other functions. But looking at the code this is only in some functions.py and the metadata_validation so it's not actually used in the code generation with reflects in no corresponding changes for these additions in the generated folder.

I'll just mark this resolved. We could look into it and removing it in the future but right now it's innocuous extra info we're not using.